lego/apps/events/tests/test_events_api.py

```diff
@@ -419,6 +481,76 @@ def test_event_creation(self):
             for key in ["name", "capacity", "activationDate", "permissionGroups"]:
                 self.assertEqual(res_pools[i][key], expect_pools[i][key])
 
+    def test_event_creation_tba(self):
+        """Test event creation for TBA status type"""
+        self.event_response = self.client.post(_get_list_url(), _test_event_data[2])
+        self.assertEqual(self.event_response.status_code, 201)
+        self.event_id = self.event_response.data.pop("id", None)
+        self.assertIsNotNone(self.event_id)
+        res_event = self.event_response.data
+        expect_event = _test_event_data[2]
+        for key in [
+            "title",
+            "description",
+            "text",
+            "start_time",
+            "end_time",
+            "merge_time",
+            "is_abakom_only",
+        ]:
+            self.assertEqual(res_event[key], expect_event[key])
+        created_event = Event.objects.get(id=self.event_id)
+        self.assertEqual(created_event.event_status_type, "tba")
+        self.assertEqual(created_event.location, "TBA")
+        self.assertEqual(len(created_event.pools.all()), 0)
+
+    def test_event_creation_open(self):
+        """Test event creation for OPEN status type"""
+        self.event_response = self.client.post(_get_list_url(), _test_event_data[3])
+        self.assertEqual(self.event_response.status_code, 201)
+        self.event_id = self.event_response.data.pop("id", None)
+        self.assertIsNotNone(self.event_id)
+        res_event = self.event_response.data
+        expect_event = _test_event_data[3]
+        for key in [
+            "title",
+            "description",
+            "text",
+            "start_time",
+            "location",
+            "end_time",
+            "merge_time",
+            "is_abakom_only",
+        ]:
+            self.assertEqual(res_event[key], expect_event[key])
+        created_event = Event.objects.get(id=self.event_id)
+        self.assertEqual(created_event.event_status_type, "open")
+        self.assertEqual(len(created_event.pools.all()), 0)
+
+    def test_event_creation_infinite(self):
```

Comment:
Should also add a test case that checks that `normal` type is used as default if no `event_status_type` is passed

Reply:
Created a test: line 573